main.py

```diff
@@ -0,0 +1,7 @@
+import callhandle as ch
+print(ch.handle_call(["VIKING 11", "VIKING 12", "VIKING 13"], ["DARKSTAR", "PYTHON 11", "PYTHON 21", "VIKING 21", "VIKING 22"])) # One Element
+print(ch.handle_call(["VIKING 11", "VIKING 12", "VIKING 21"], ["DARKSTAR", "PYTHON 11", "PYTHON 21", "VIKING 21", "VIKING 22"])) # Two element
+print(ch.handle_call(["VIKING 11", "VIKING 12", "PYTHON 21"], ["DARKSTAR", "PYTHON 11", "PYTHON 21", "VIKING 21", "VIKING 22"])) # Not all same flight
+print(ch._find_flight_name(["VIKING 11", "VIKING 12", "VIKING 13"]))
+print(ch._check_same_flight(["VIKING 11", "VIKING 12", "VIKING 13"], "VIKING"))
+print(ch._check_same_flight(["VIKING 11", "VIKING 12", "PYTHON 13"], "VIKING"))
```

Comment:
Generally you shouldn't really be testing `__private`/`_protected` (most teams use a single underscore for both private/protected though) methods, either change them to `public` or don't test them.

Reply:
I actually meant to remove those tests, they were just for while I was writing those methods